### PR TITLE
Broken image links in PushNotifications-spec.md

### LINKED ISSUE #944: Broken image link in PushNotifications-spec.md
**Describe the bug**
Broken image links in [PushNotifications-spec.md](https://github.com/microsoft/ProjectReunion/blob/main/specs/PushNotifications/PushNotifications-spec.md)

### PR DESCRIPTION
**Pull request by shikharvashistha**
Fixed Broken image links in [PushNotifications-spec.md](https://github.com/shikharvashistha/ProjectReunion/blob/user/shikhar/fixed-broken-links/specs/PushNotifications/PushNotifications-spec.md)

[Before](https://github.com/microsoft/ProjectReunion/blob/main/specs/PushNotifications/PushNotifications-spec.md) -> [Now](https://github.com/shikharvashistha/ProjectReunion/blob/user/shikhar/fixed-broken-links/specs/PushNotifications/PushNotifications-spec.md)

Closes #944

### CODE REVIEW

**Comment by shikharvashistha** [comment]
@huichen123 @axelandrejs 

Kindly review the PR and suggest changes/improvements if any.

File: specs/PushNotifications/PushNotifications-spec.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ events that require immediate user/developer action.
 * [The Windows Push Notification Services (WNS) Overview](https://docs.microsoft.com/windows/uwp/design/shell/tiles-and-notifications/windows-push-notification-services--wns--overview) Defines how an app today can subscribe to Push Notifications and send a payload down to the corresponding client.
 
 The below diagram is an illustration of the overall Push Flow that we have today:<br/>
-![Legacy Flow For Project Reunion](Legacy.png)
+![Legacy Flow For Project Reunion](Legacy.PNG)
```
Comment (hulumane) [inline specs/PushNotifications/PushNotifications-spec.md:16]: Thanks for fixing this. It's interesting that the linked files are case-sensitive!

**Review by hulumane — APPROVED** [review]

**Review by huichen123 — APPROVED** [review]
Thanks for fixing the broken links.

**Comment by DrusTheAxe** [comment]
Is this obsolete?

**Comment by shikharvashistha** [comment]
> Is this obsolete?

Greetings of the day @DrusTheAxe ,

I believe they forgot to merge this.

Thanks and Regards
@shikharvashistha

**Review by raghavthind2005 — APPROVED** [review]

**Comment by bpulliam** [comment]
The correct change was eventually made.  Sorry this got dropped on the floor, but thanks for the effort!

**Pull request closed without merging**